### PR TITLE
Recursive cran functionality added to bioconductor-skeleton.py 

### PR DESCRIPTION
**Pull request by MathiasHaudgaard**
Elixir in Aarhus has added a feature to bioconductor-skeleton so if there are any cran dependencies, there will be made recipes for these dependencies. However, we still have the following problems:
- What if you want to make the same recipe just with a different version. Right now that situation results in an error because the script sees that the recipe is already made.
- Not sure if we're handling r-base correct.

Feel free to comment and give suggestions :-)

### CODE REVIEW

File: maintainers.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+extra:
+   recipe-maintainers:
```
Comment (jdblischak) [inline maintainers.yaml]: Are the GitHub usernames case sensitive?

**Comment by jdblischak** [comment]
> What if you want to make the same recipe just with a different version. Right now that situation results in an error because the script sees that the recipe is already made.

@MathiasHaudgaard Are you referring to the original recipe or the dependencies? What happens when you set `force = True` for `write_recipe`?

File: maintainers.yaml (hunk 1 above)
Comment (bgruening) [reply maintainers.yaml]: I think so too.

Comment (MathiasHaudgaard) [reply maintainers.yaml]: I'll change that :-)

File: bioconda_utils/cran_skeleton.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
+from optparse import OptionParser
```
Comment (bgruening) [inline bioconda_utils/cran_skeleton.py]: Can we use argparse here?

File: bioconda_utils/cran_skeleton.py [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+import subprocess as sp
+import re
+from itertools import zip_longest
+import argparse
+import yaml
+
+
```
Comment (bgruening) [inline bioconda_utils/cran_skeleton.py]: Can we add a short description about the purpose of this file?
This is the file which you started at the ELIXIR hackathon in Paris, right? The port of conda_r_helper_script?

If so we could just add, that this file cleans-up normal cran skeletons to make it conda-forge compliant.

**Comment by bgruening** [comment]
@jdblischak @daler should we add some tests before merging?

**Comment by daler** [comment]
@bgruening @jdblischak tests would be good, I'll work on that.

**Comment by daler** [comment]
Closed via #242

**Pull request closed without merging**